test/integration_test.rb

```diff
@@ -44,6 +44,31 @@ def test_tests_are_selected_based_on_changes_in_current_working_tree
         output = rake('ttnt:test:run')
         assert_match '2 runs, 2 assertions, 2 failures', output[:stdout]
       end
+
+      def test_storage_file_resides_with_rakefile
+        tmpdir = 'tmp'
```

Comment:
You can certainly directly put `'tmp'` instead of creating a variable for that ; the name is longer and you have to interpolate the value later on.